roles/aodh/tasks/main.yml

```diff
@@ -27,6 +28,17 @@
     - aodh-evaluator
     - aodh-listener
     - aodh-notifier
+  when: os == 'ubuntu'
+
+- name: install aodh services (rhel)
+  yum: pkg={{ item }} state=installed
+  with_items:
+    - "{{ openstack_meta.aodh.services.aodh_api[os].name }}"
+    - "{{ openstack_meta.aodh.services.aodh_evaluator[os].name }}"
+    - "{{ openstack_meta.aodh.services.aodh_listener[os].name }}"
+    - "{{ openstack_meta.aodh.services.aodh_notifier[os].name }}"
+    - "{{ openstack_meta.aodh.services.aodh_expirer[os].name }}"
+  when: os == 'rhel'
```

Comment:
Also, need to add a task to add firewalld rules. See nova-control or any other core services for example. Use block with when: os == 'rhel'